vite.config.ts

```diff
@@ -1,11 +1,29 @@
-import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import { VitePWA } from "vite-plugin-pwa";
 import viteTsconfigPaths from 'vite-tsconfig-paths';
-import svgrPlugin from 'vite-plugin-svgr';
```

Comment:
This got removed?

Reply:
We don't have any SVG files for now, so I just remove import, not the package